app/src/main/java/com/hillnerds/soundsparrow/Sound.java

```diff
@@ -18,30 +19,26 @@
 
 public class Sound extends AppCompatActivity implements MidiDriver.OnMidiStartListener {
     protected MidiDriver midi;
-    protected MediaPlayer player;
 
     protected long userId = MainActivity.uuid_long;
-    protected long randomSeed = userId;
+    //protected long randomSeed = userId;
```

Comment:
Again, commented out code. This code exists in the previous commit, so if you really needed it again, you could pull it from there.